meshroom/ui/qml/Viewer3D/Inspector3D.qml

```diff
@@ -298,10 +298,11 @@ FloatingPane {
 
                 SearchBar {
                     id: searchBar
-                    Layout.minimumWidth: 150
-                    Layout.fillWidth: true
-                    Layout.rightMargin: 10
-                    Layout.leftMargin: 10
+
+                    toggle: true // enable toggling the actual text field by the search button
```

Comment:
```suggestion
                    toggle: true  // enable toggling the actual text field by the search button
                    isVisible: true
```

I think the search bar should already be expanded in the case of the 3D inspector. Otherwise, you just have the magnifier icon on the top-left that blocks the entire row while the loaded elements are displayed on the entire row below, and it creates a major imbalance visually speaking.

Reply:
I also think it'd be better to have some left- and right-margins in this case: the search bar takes up 100% of the horizontal space, which is not the case of the loaded 3D elements, and it also creates an imbalance.

![Screenshot_2024-11-12_10-33-26](https://github.com/user-attachments/assets/3608a218-af7d-4110-b66b-a7dfae49a0bc)